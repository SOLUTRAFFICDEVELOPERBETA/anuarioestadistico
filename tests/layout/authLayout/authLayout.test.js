import React from 'react';
import '@testing-library/jest-dom';
import { mount } from 'enzyme';
import AuthLayout from '../../../Layout/authLayout';
import ThemeContext from '../../../contexts/theme';

describe('Pruebas en AuthLayout', () => {
    const children = (<p><b>Dios</b> es amor</p>)
    const bg = {
        name: 'imagen prueba',
        path: '/img/prueba.png',
        url: 'https://www.valladolid.mx/assets/img/default.jpg'
    }
    const wrapper = mount(
        <ThemeContext.Provider value={{
            bg
        }}>
            <AuthLayout
                children={children}
            />
        </ThemeContext.Provider>
    )

    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('debe de mostrar la informaciÃ³n del children', () => {
        expect(wrapper.find('p').html()).toBe('<p><b>Dios</b> es amor</p>')
    })
})
