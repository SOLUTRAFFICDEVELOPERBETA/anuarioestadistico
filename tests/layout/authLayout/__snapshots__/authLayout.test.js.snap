// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en AuthLayout debe mostrarse correctamente 1`] = `
<div
  className="css-6ctbyo-ContainerDiv e15g2ioh0"
>
  <p>
    <b>
      Dios
    </b>
     es amor
  </p>
</div>
`;
